.eslintrc.js

```diff
@@ -109,6 +109,12 @@ module.exports = getESLintConfig({
           'import/named': 0,
           'no-new': 0
         }
+      },
+      {
+        files: ['modules/widgets/**/*.tsx'],
+        rules: {
+          'react/react-in-jsx-scope': 0
```

Comment:
It would be very helpful to add comments here, future maintainers will likely struggle to understand the context. E.g: I am assuming we are not actually using react but using these rules with preact?

Reply:
Added "Disable React-style JSX linting since they conflict with Preact JSX."